tasks/tasks_test.go

```diff
@@ -0,0 +1,67 @@
+package tasks
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTaskErrorLeadsToErrorStatus(t *testing.T) {
```

Comment:
since uppercase and thus exportable, shouldn't Go complain about lack of comment?